tripFrontend/src/features/product/components/ProductForm/ProductForm.tsx

```
@@ -0,0 +1,368 @@
+	form: UseFormReturn<TRequestBodyCreateProduct>;
+	onSubmit?: SubmitHandler<TRequestBodyCreateProduct>;
+	onCancel?: () => void;
+};
+
+const Required = (): JSX.Element => (
+	<span className="text-red-600" aria-hidden>
+		*
+	</span>
+);
+
+const Error = ({ id, errors }: TError): JSX.Element => (
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)